contracts/test/zapper/oethb-zapper.base.fork-test.js

```diff
@@ -0,0 +1,91 @@
+const { createFixtureLoader } = require("../_fixture");
+const { defaultBaseFixture } = require("../_fixture-base");
+const { expect } = require("chai");
+const { oethUnits } = require("../helpers");
+
+const baseFixture = createFixtureLoader(defaultBaseFixture);
+
+describe("ForkTest: OETHb Zapper", function () {
+  let fixture;
+  beforeEach(async () => {
+    fixture = await baseFixture();
+  });
+
+  it("Should mint OETHb with ETH", async () => {
```

Comment:
we should also add a test where ETH is simply sent to the zapper without calling any function